tests/integration/sqcmds/vmx-samples/swport.yml

```diff
@@ -44,5 +44,5 @@ tests:
 - command: interface show --columns=vlan --vlan='101 201' --namespace=vmx --format=json
   data-directory: tests/data/parquet/
   marks: interface show vmx switchport
-  output: '[{"vlan": 0}, {"vlan": 101}, {"vlan": 0}, {"vlan": 101}, {"vlan": 0}, {"vlan":
-    201}, {"vlan": 0}, {"vlan": 201}]'
+  output: '[{"vlan": 101}, {"vlan": 101}, {"vlan": 0}, {"vlan": 201}, {"vlan": 0},
+    {"vlan": 201}]'
```

Comment:
why vlan `0` is still in the output?

Reply:
Because we look to both `vlan` and `vlanList`, in this case vlanList contains `[101, 201]`